src/includes/getting-started-primer/javascript.cordova.mdx

```diff
@@ -0,0 +1,19 @@
+<Note>
+
+_Sentry's Cordova SDK enables automatic reporting of errors, messages, and exceptions, and identifies performance issues in your application. The SDK is available on GitHub [`sentry-cordova`](https://github.com/getsentry/sentry-cordova)._
+
+_The Cordova SDK uses a native extension for iOS and Android, but will fall back to a pure JavaScript version (`@sentry/browser`) if needed. When you install `sentry-cordova`, the [Sentry Wizard](https://github.com/getsentry/sentry-wizard) will help configure your project. We also add a build step to your Xcode project to upload debug symbols we need to symbolicate iOS crashes._
+
+</Note>
+
+Features:
+
+- Automatic Native Crash [Error Tracking](/product/issues/) using both [iOS](/platforms/apple/guides/ios/) and [Android](/platforms/android/)
+- Offline storage of events
+- Events [enriched](/platforms/javascript/guides/cordova/enriching-events/context/) with device data
+- [Breadcrumbs](/platforms/javascript/guides/cordova/enriching-events/breadcrumbs/) created for location changes
+- On-device symbolication for JavaScript (in `Debug` mode)
+- [Ionic support](/platforms/javascript/guides/cordova/ionic/)
+- Under the hood the SDK relies on our [JavaScript SDK](/platforms/javascript/), which makes all functions available for JavaScript also available in this SDK including:
+    - [Performance Monitoring](/product/performance/) to create transactions
+    - [Release Health](/product/releases/health/) to track crash free users and sessions
```

Comment:
release health is supported thru the Native SDK and not the JavaScript SDK.
should we move it under `Automatic Native Crash`?